### ISSUE TITLE
Missing Kafka libraries in Docker v1.10.8 image

### ISSUE THREAD

**Issue by JeroenMinnaert** [zendesk/maxwell#791]
Can you please update the official Docker image so it includes other Kafka libraries then kafka-clients-0.9.0.1.jar?

**Comment by xuebingli** [comment]
@liulikun bug confirmed. 
```
$ docker run -it zendesk/maxwell:v1.10.6 bash
root@837d9f77d403:/app# ls lib | grep "kafka"
kafka-clients-0.10.0.1.jar
kafka-clients-0.10.1.0.jar
kafka-clients-0.10.2.1.jar
kafka-clients-0.8.2.2.jar
kafka-clients-0.9.0.1.jar
root@837d9f77d403:/app# exit
$ docker run -it zendesk/maxwell:v1.10.8 bash
root@e975a8ed6e80:/app# ls lib | grep "kafka"
kafka-clients
root@e975a8ed6e80:/app# ls lib/kafka-clients
kafka-clients-0.9.0.1.jar
root@e975a8ed6e80:/app# exit
```

**Comment by xuebingli** [comment]
The bug is likely introduced in this change https://github.com/zendesk/maxwell/commit/abd348088271d2df20c817d48a5f516a50a4276d

**Comment by xuebingli** [comment]
Fixed in [v1.10.9](https://github.com/zendesk/maxwell/releases/tag/v1.10.9).
